py-polars/polars/internals/series.py

```diff
@@ -581,6 +591,8 @@ def log10(self) -> "Series":
         """
         Return the base 10 logarithm of the input array, element-wise.
         """
+        if not _NUMPY_AVAILABLE:
```

Comment:
This can be implemented directly by polars: ```difff
@@ -581,7 +581,7 @@ class Series:
         """
         Return the base 10 logarithm of the input array, element-wise.
         """
-        return np.log10(self)  # type: ignore
+        return self.log(10.0)
```

Reply:
Nice, we can fix this one right away! Do you know if there are similar 'quick fixes' among the other numpy functionality?